usecases/token/generator.go

```diff
@@ -60,7 +60,7 @@ func (g *Generator) fromAPIKey(ctx context.Context, apiKey string) (string, time
 			fmt.Errorf("GetOrganizationByID error: %w", err)
 	}
 
-	name := fmt.Sprintf("ApiKey Of %s", organization.Name)
+	name := fmt.Sprintf("Api key %s*** of %s", key.Prefix, organization.Name)
```

Comment:
IIUC, the Prefix is used by convenience :
- to track which api key is used
- to communicate with a client, in case we need to investigate issues about a perticular apikey (= impossible to use postgress ID for that)

Is the name prop used to create event ?